SkinJSON.php

```diff
@@ -137,13 +139,13 @@ private static function hookTestElement( string $hook, Config $config, $inline =
 		}
 	}
 
-	public static function onSkinAfterPortlet( $skin, $name, &$html ) {
+	public static function onSkinAfterPortlet( $skin, $portlet, &$html ) {
```

Comment:
I think using portletName makes the most sense here.
I've pushed an upstream change: https://gerrit.wikimedia.org/r/c/mediawiki/core/+/812379